sound/soc/sof/intel/hda-pcm.c

```diff
@@ -118,7 +118,7 @@ int hda_dsp_pcm_hw_params(struct snd_sof_dev *sdev,
 
 	/* set host_period_bytes to 0 if no IPC position */
 	if (hda && hda->no_ipc_position)
-		ipc_params->host_period_bytes = 0;
+		ipc_params->period_irq = 0;
```

Comment:
Also the field is used before it's added to the header file in the next commit, this would break git bisect. Don't do it.

Reply:
Yes, good catch thanks.